dandi/files/bases.py

```diff
@@ -526,9 +537,10 @@ def get_validation_errors(
         else:
             # make sure that we have some basic metadata fields we require
             try:
-                origin = ValidationOrigin(
-                    name="nwbinspector",
-                    version=str(_get_nwb_inspector_version()),
+                origin = Origin(
```

Comment:
```suggestion
                origin_nwbinspector = Origin(
```

Reply:
Renamed to `origin_validation_nwbinspector` to be consistent with other names.